openid-4-verifiable-presentations-1_0.md

```diff
@@ -1534,6 +1531,45 @@ issuers in Self-Sovereign Identity ecosystems using TRAIN</title>
         </front>
 </reference>
 
+<reference anchor="IANA.OAuth.Parameters" target="https://www.iana.org/assignments/oauth-parameters">
+  <front>
+    <title>OAuth Parameters</title>
+    <author>
+      <organization>IANA</organization>
+    </author>
+    <date/>
+  </front>
+</reference>
+
+<reference anchor="IANA.MediaTypes" target="https://www.iana.org/assignments/media-types">
+  <front>
+    <title>Media Types</title>
+    <author>
+      <organization>IANA</organization>
+    </author>
+    <date/>
+  </front>
+</reference>
+
+<reference anchor="IANA.URI.Schemes" target="https://www.iana.org/assignments/uri-schemes">
+  <front>
+    <title>Uniform Resource Identifier (URI) Schemes</title>
+    <author>
+      <organization>IANA</organization>
+    </author>
+    <date/>
+  </front>
+</reference>
+
+<reference anchor="IANA.JOSE" target="https://www.iana.org/assignments/jose">
+        <front>
+          <title>JSON Object Signing and Encryption (JOSE)</title>
+          <author>
+            <organization>IANA</organization>
+          </author>
+        </front>
+</reference>
+
```

Comment:
```suggestion

<reference anchor="IANA.JOSE" target="https://www.iana.org/assignments/named-information/named-information.xhtml">
        <front>
          <title>Named Information Hash Algorithm</title>
          <author>
            <organization>IANA</organization>
          </author>
        </front>
</reference>

```

per this comment 
https://github.com/openid/OpenID4VP/pull/197/files#diff-3118c9756a1d7b361bb53af8bd9d65666476a27cfeeced56c47a5dccc38eac55R291

Reply:
I talked with Kristina about this and actually we can do this in her PR so marking this as resolved.